twarep/electrifly
# Power scatter plot

## Checklist
**Note:** Put an x into the box like this [x] to check these off

## What I Did
What files git you edit and why/for what purpose?
- app.py to add the power vs time scatter plot
<img width="1679" alt="Screen Shot 2023-08-05 at 12 22 14 PM" src="https://github.com/twarep/electrifly/assets/71115362/9c950ac8-5ab6-4614-ade0-1773dfdb67a8">


## How I Did it?
What packages did I install to accomplish this goal? What libraries did I need that the team might not know?
Write anything in this section that is technically related that everyone on the team might not know or is not
trivial.
- followed similar pattern from the other UI graphs

## How to Test?
Manual testing? Automated testing? E2E testing? Smoke tests?
- manual testing (I used the test DB data but should work in both prod and test environments)
- make sure you can select multiple flights

### Relevant Links
Any tutorials or other resources used to help with this task. You can also cite the documentation.
